stories/multi-combo-box/multi-combo-box.stories.js

```diff
@@ -0,0 +1,1023 @@
+        </div>
+        </div>
+    </div>
+    </div>
+</div>
+`;
+
+asFormItem.parameters = {
+    docs: {
+        storyDescription: `
+Note that the popover body width is restricted to a max of 37.5rem to avoid readability issues in large-width popovers.
+Applications are free to override this in their custom styles if needed and own any readability issues arising from this override.
+`
+    }
+};
+
+export const grouping = () => `<div style="height: 400px;">
+    <div class="fd-popover">
+        <div class="fd-popover__control" role="combobox" aria-controls="F4H8X34a" aria-expanded="true" aria-haspopup="true">
+            <div class="fd-input-group fd-input-group--control"  tabindex="0">
+                <div class="fd-tokenizer">
+                    <div class="fd-tokenizer__inner">
+                        <span class="fd-token" role="button" tabindex="0">
+                            <span class="fd-token__text">
+                                Apple
+                            </span>
+                            <button class="fd-token__close" aria-label="close button"></button>
+                        </span>
+                        <span class="fd-token" role="button" tabindex="0">
+                            <span class="fd-token__text">
+                                Orange
+                            </span>
+                            <button class="fd-token__close" aria-label="close button"></button>
+                        </span>
+                        <span class="fd-token" role="button" tabindex="0">
+                            <span class="fd-token__text">
+                                Kiwi
+                            </span>
+                            <button class="fd-token__close" aria-label="close button"></button>
+                        </span>
+                        <span class="fd-tokenizer__indicator">2 more</span> 
+                        <input class="fd-input fd-input-group__input fd-tokenizer__input" aria-label="multi input sample as group item" />
+                    </div>
+                </div>
+                <span class="fd-input-group__addon fd-input-group__addon--button">
+                    <button class="fd-input-group__button fd-button fd-button--transparent" aria-label="down arrow" aria-controls="F4H8X34a" aria-expanded="false" aria-haspopup="true">
+                        <i class="sap-icon--navigation-down-arrow"></i>
+                    </button>
+                </span>
+            </div>
+        </div>
+        <div class="fd-popover__body fd-popover__body--no-arrow fd-popover__body--dropdown fd-popover__body--dropdown-fill" aria-hidden="false" id="F4H8X34a">
+        <div class="fd-popover__wrapper">
+           <label class="fd-list__group-header" id="selectMultipleFruitsLabel">
+                Fruits
+            </label>
+            <ul class="fd-list fd-list--multi-input" role="listbox" aria-multiselectable="true" aria-label="selectMultipleFruitsLabel">
```

Comment:
`aria-label` should instead be `aria-labelledby` as it is referring to the group header's `id` above it. Similar to what is done for Vegetables, which is correct.
```suggestion
            <ul class="fd-list fd-list--multi-input" role="listbox" aria-multiselectable="true" aria-labelledby="selectMultipleFruitsLabel">
```

Reply:
Expected the HTML found at $('ul[aria-labelledby="Fruits"]') to have no violations:

    <ul class="fd-list fd-list--multi-input" role="listbox" aria-multiselectable="true" aria-labelledby="Fruits">

    Received:

    "ARIA attributes must conform to valid values (aria-valid-attr-value)"

    Fix all of the following:
      Invalid ARIA attribute value: aria-labelledby="Fruits"
@jacobdevera  i get this error so i have changed it to aria-label

Reply:
Rather than `aria-labelledby="Fruits"`, it should be `aria-labelledby="selectMultipleFruitsLabel"` as noted in the suggestion. That is the `id` that refers to the `<label>` element which contains the "Fruits" text. `aria-labelledby` can only contain ids, `aria-label` should only contain actual text which is used as a label when there is no visible `<label>` element to refer to.